### PR TITLE
New page for 2022-01-31 portal release note

### PR DESCRIPTION
**Pull request by sajumb**

### CODE REVIEW

File: Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+The blob storage connection info for a project was previously only displayed in Kudu which was not very convenient or easy to find. A developer should spend as much time as possible developing fantastic solutions and less time in Kudu.
+Therefore this connection info is now easy to **_copy directly from the portal_** and ready to post into _Azure Blob Explorer_ whenever needed.
+
+![BlobStorage](https://user-images.githubusercontent.com/93588665/151602205-2784ec6c-1142-4221-9bf4-0ba9727ff8f6.gif)
+
+### Project invitation link to clipboard
+The project invitation flow in Umbraco Cloud Portal has until recently suffered from invites ending up in the invitees spam-folder. We have now **_optimized_** the configuration of the portal _**email delivery**_ so every invite should now be delivered to the expected receiver - in the correct inbox.
+
+However, there can be exceptions where an email is either bounced or lands in the spam folder. In such cases a resend of the project invitation might not do the trick. So to ease the invite process, in the rare case the email doesn’t show up, we have added the option to **_copy the project invitation link_** for active invites.
```
Comment (jonat123) [inline Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md:19]:
```suggestion
### Project invitation link to clipboard
The project invitation flow in Umbraco Cloud Portal has until recently suffered from invites ending up in the invitees' spam folder. We have now **_optimized_** the configuration of the portal _**email delivery**_ so every invites should now be delivered to the expected receiver - in the correct inbox.

However, there can be exceptions where an email is either bounced or lands in the spam folder. In such cases, a resend of the project invitation might not do the trick. So to ease the invite process, in the rare case the email doesn’t show up, we have added the option to **_copy the project invitation link_** for active invites.
```

File: Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+![CopyProjectInviteLink](https://user-images.githubusercontent.com/93588665/151602357-1bd4b165-eb4d-44b5-bc88-b45594ae5dc0.gif)
+
+### Release notes link in portal
+In 2021 we added several new team members to the Cloud Feature team to support our strategy of making Umbraco Cloud [the best way](https://umbraco.com/blog/umbraco-2022-and-onwards/) to host Umbraco solutions. During 2022 and the years to come we will continue our _**customer-centric development**_ and with an increased focus on  input from partners and portal users improve existing functionality and create new exciting features.
+
+In order for agencies and users to keep up with the improvements, we’ve made it easy to find the latest release notes. You can now find the link for the release notes in the profile settings menu. Release notes will be published multiple times each month and list the most relevant fixes and features added to the portal.
```
Comment (jonat123) [inline Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md:26]:
```suggestion
### Release notes link in the portal
In 2021 we added several new team members to the Cloud Feature team to support our strategy of making Umbraco Cloud [the best way](https://umbraco.com/blog/umbraco-2022-and-onwards/) to host Umbraco solutions. In 2022 and the years to come we will continue our _**customer-centric development**_ and with an increased focus on input from partners and portal users improve existing functionality and create new exciting features.

In order for agencies and users to keep up with the improvements, we’ve made it easy to find the latest release notes. You can now find the link for the release notes in the profile settings menu. Release notes will be published multiple times each month and list the most relevant fixes and features added to the portal. 
```

**Comment by jonat123** [comment]
@sajumb Looks good!

I just went over it quickly and made some suggestions, let me know when they are added and I'll merge it in right away :D

File: Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md (hunk 1 above)
Comment (sajumb) [reply Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md:19]: Hi @jonat123,
Rune has already reviewed this, but feel free to change what he find relevant. (however I do believe that it should be 'invite' and note 'invites' in plurum)

File: Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md (hunk 2 above)
Comment (sajumb) [reply Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md:26]: Yup, the suggestions for this are fine to merge.

File: Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md (hunk 1 above)
Comment (jonat123) [reply Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md:19]: Ahh fair, i'll change the invite  part back :)

Comment (jonat123) [inline Umbraco-Cloud/release-notes/2022-01-31-ReleaseNotes.md:19]:
```suggestion
The project invitation flow in Umbraco Cloud Portal has until recently suffered from invites ending up in the invitees' spam folder. We have now **_optimized_** the configuration of the portal _**email delivery**_ so every invite should now be delivered to the expected receiver - in the correct inbox.

However, there can be exceptions where an email is either bounced or lands in the spam folder. In such cases, a resend of the project invitation might not do the trick. So to ease the invite process, in the rare case the email doesn’t show up, we have added the option to **_copy the project invitation link_** for active invites.
```

**Comment by jonat123** [comment]
Merging in this PR :-)

**Pull request merged**